includes_php56/class-wcmp-frontend.php

```diff
@@ -145,7 +145,7 @@ public function wpo_wcpdf_delivery_options($replacement, $order)
 
             return ob_get_clean();
         }
-        // options.delivery_date custom delivery date not supported for carrier bpost
+        // options.delivery_date custom delivery date not supported for carrier postnl
```

Comment:
Dit is toch niet zo voor postnl?

Reply:
de map includes_php56 gaat helemaal vervangen worden, alleen moet dit binnen het team besproken worden. Maar omdat niemand er nu is heb ik alleen de bpost naar postnl veranderd